core/trino-web-ui/src/main/resources/webapp/vendor/vis/vis.css

```diff
@@ -0,0 +1,810 @@
+.vis .overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
```

Comment:
Assuming this is a custom style sheet , remove a the empty lines within each class